web/struct_tree_layer_builder.js

```diff
@@ -0,0 +1,148 @@
+/* Copyright 2021 Mozilla Foundation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+const PDF_ROLE_TO_HTML_ROLE = {
```

Comment:
This looks *very* similar to the list in `src/display/struct_tree_layer.js`, are they in fact identical?
If so, then please export that one in the official API (via `src/pdf.js`) rather than duplicating this here.

Reply:
Loooks like I forgot to git rm display/struct_tree_layer.js.